### ISSUE TITLE
As an Applicant, I would like to be able to contact a help desk if I have difficulty with the online system and be able to contact a SME if I have questions about how to complete the application.

### ISSUE THREAD

**Issue by EStriegel** [18F/dol-whd-14c#156]

**Comment by mgwalker** [comment]
Does WHD have a help desk number, email box, etc. already?

**Comment by oghaffari** [comment]
Specifically about how to fill out the application, not technical assistance.

**Comment by phirefly** [comment]
@EStriegel What do you think about marking this as `blocked` or something similar to indicate that we need a business decision on this?

**Comment by EStriegel** [comment]
@phirefly That's fine with me, but just to clarify is this more of a IT business decision?  There's a general certification team number that can be posted for assistance with "how to fill out the form" questions, but we do not have a mechanism for technical assistance.  Once we have that, it seems more like a user experience question of how we make both available and clearly communicate the difference between the two types of assistance available?

**Comment by mmurthydol** [comment]
@phirefly @estriegel In my understanding the scope of this story is about help filling out the application. If we are providing a phone number (once business decision is reached) for assistance filling out the form, this story can be unblocked for the specific purpose, while providing specs (phone #, verbiage, Ux workflow/page).  

IT Technical assistance/Application Support issues (application errors, browser troubleshooting, performance, etc.): This should be a separate story here once we know what we want is vendor to develop for error handling, verbiage, help desk etc.  DOL needs an internal story as well to decide on Application Support issues.

**Comment by EStriegel** [comment]
Pending resolution re: WHD help desk issue.

**Comment by phirefly** [comment]
@phirefly and @EStriegel will add text to the story including elements 1) error message and 2) number to call

**Comment by EStriegel** [comment]
@simplyshang Here is the language for the SME contact info - 

[Help Page - SME Contact Info.docx](https://github.com/18F/dol-whd-14c/files/1511292/Help.Page.-.SME.Contact.Info.docx)

**Comment by mmurthydol** [comment]
@EStriegel Liz, this info looks great for business and not technical support issues! 
@phirefly For application technical support issues - are we looking at using an email/phone number combo? Wondering what's the word on that - thought you were scoping this issue out. Thanks.
